docs/en/operations/system_tables.md

```diff
@@ -252,55 +252,81 @@ This is similar to the DUAL table found in other DBMSs.
 
-- rows (UInt64) – The number of rows.
+- `rows` (`UInt64`) – The number of rows.
 
-- bytes (UInt64) – The number of bytes when compressed.
+- `bytes_on_disk` (`UInt64`) – Total size of all the data part files in bytes.
 
-- modification_time (DateTime) – The modification time of the directory with the data part. This usually corresponds to the time of data part creation.|
+- `data_compressed_bytes` (`UInt64`) – Total size of compressed data in the data part. All the auxiliary files (for example, files with marks) are not included.
 
-- remove_time (DateTime) – The time when the data part became inactive.
+- `data_uncompressed_bytes` (`UInt64`) – Total size of uncompressed data in the data part. All the auxiliary files (for example, files with marks) are not included.
 
-- refcount (UInt32) – The number of places where the data part is used. A value greater than 2 indicates that the data part is used in queries or merges.
+- `marks_bytes` (`UInt64`) – The size of the file with marks.
 
-- min_date (Date) – The minimum value of the date key in the data part.
+- `modification_time` (`DateTime`) – The modification time of the directory with the data part. This usually corresponds to the time of data part creation.|
 
-- max_date (Date) – The maximum value of the date key in the data part.
+- `remove_time` (`DateTime`) – The time when the data part became inactive.
 
-- min_block_number (UInt64) – The minimum number of data parts that make up the current part after merging.
+- `refcount` (`UInt32`) – The number of places where the data part is used. A value greater than 2 indicates that the data part is used in queries or merges.
 
-- max_block_number (UInt64) – The maximum number of data parts that make up the current part after merging.
+- `min_date` (`Date`) – The minimum value of the date key in the data part.
 
-- level (UInt32) – Depth of the merge tree. If a merge was not performed, ``level=0``.
+- `max_date` (`Date`) – The maximum value of the date key in the data part.
 
-- primary_key_bytes_in_memory (UInt64) – The amount of memory (in bytes) used by primary key values.
+- `min_time` (`DateTime`) – The minimum value of the date and time key in the data part.
 
-- primary_key_bytes_in_memory_allocated (UInt64) – The amount of memory (in bytes) reserved for primary key values.
+- `max_time`(`DateTime`) – The maximum value of the date and time key in the data part.
 
-- database (String) – Name of the database.
+- `partition_id` (`String`) – Id of the partition.
 
-- table (String) – Name of the table.
+- `min_block_number` (`UInt64`) – The minimum number of data parts that make up the current part after merging.
 
-- engine (String) – Name of the table engine without parameters.
+- `max_block_number` (`UInt64`) – The maximum number of data parts that make up the current part after merging.
+
+- `level` (`UInt32`) – Depth of the merge tree. If a merge was not performed, `level=0`.
```

Comment:
```suggestion
- `level` (`UInt32`) – Depth of the merge tree. Zero means that current part was created by insert rather than by merging other parts.
```